app/Http/Controllers/UserAdminController.php

```diff
@@ -54,7 +54,9 @@ public function index(Request $request)
     public function details($id)
     {
         $user = User::findOrFail($id);
-        return view('users.admin.details', ['user' => $user]);
+        $memberships['pending'] = Member::withTrashed()->where('user_id', '=', $user->id)->where('deleted_at', '=', null)->where('pending', '=', true)->get();
+        $memberships['previous'] = Member::withTrashed()->where('user_id', '=', $user->id)->where('deleted_at', '!=', null)->get();
+        return view('users.admin.details', ['user' => $user, 'memberships' => $memberships]);
```

Comment:
It shouldn't happen, but what if a user has multiple non pending non deleted memberships, then only one is shown.

Reply:
Should I just add another line for "broken" memberships? 😂